Makefile

```diff
@@ -1,77 +1,7 @@
-.PHONY: all install clean doc github test cdtest cdtest_tls
-
-EXT_OBJ:=$(shell ocamlfind ocamlc -config | awk '/^ext_obj:/ {print $$2}')
-EXT_LIB:=$(shell ocamlfind ocamlc -config | awk '/^ext_lib:/ {print $$2}')
-
-OCAMLBUILD = ocamlbuild -use-ocamlfind -classic-display \
-	-cflags "-w A-4-33-40-41-42-43-34-44" \
-	-plugin-tag "package(ppx_driver.ocamlbuild)"
-
-PREFIX ?= /usr/local/bin
-OS_TYPE:=$(shell ocamlfind ocamlc -config | awk '/^os_type:/ {print $$2}')
-ifeq ($(OS_TYPE),$(filter $(OS_TYPE),Win32 Cygwin))
-EXT_EXE=.exe
-else
-EXT_EXE=
-endif
-
-
-B=_build/lib
-FILES = $(wildcard $B/*.cmi $B/*.cmt $B/*.cmti $B/*.cmx $B/*.cmxa $B/*.cma $B/*.cmxs $B/*$(EXT_LIB) $B/*$(EXT_OBJ) $B/*.cmo)
-MORE_FILES = $(wildcard lib/intro.html $B/*.mli)
+.PHONY: all clean
 
 all:
-	$(OCAMLBUILD) conduit.otarget
-
-install:
-	rm -rf _install
-	mkdir -p _install
-ifneq ("$(wildcard _build/lib/conduit_xenstore.cmo)","")
-	echo '"scripts/xenstore-conduit-init" {"xenstore-conduit-init"}' > _install/bin
-endif
-	$(foreach f,$(FILES), echo "$(f)" >> _install/lib;)
-	ocamlfind remove conduit || true
-	ocamlfind install conduit META $(FILES) $(MORE_FILES)
+	jbuilder build
 
 clean:
-	$(OCAMLBUILD) -clean
-	rm -rf _install lib/conduit_config.mlh META _tags
-
-doc:
-	$(OCAMLBUILD) lib/conduit.docdir/index.html
-
-github: doc
-	git checkout gh-pages
-	git merge master --no-edit
-	$(MAKE)
-	rm -f *.html
-	cp _build/lib/conduit-all.docdir/* .
-	git add *.html
-	cp nice-style.css style.css
-	git add style.css
-	git commit -m 'sync ocamldoc' *.html *.css
-	git push
-	git checkout master
-
-VERSION = $(shell cat VERSION)
-NAME    = conduit
-ARCHIVE = https://github.com/mirage/ocaml-$(NAME)/archive/v$(VERSION).tar.gz
-
-release:
-	git tag -a v$(VERSION) -m "Version $(VERSION)."
-	git push upstream v$(VERSION)
-	$(MAKE) pr
-
```

Comment:
yea, updating the mirage.io instructions is on my todo list, I'll try to do this before the end of the week